README.md

```diff
@@ -37,7 +37,7 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
  [Microsoft PowerToys GitHub releases page][github-release-link], click on `Assets` at the bottom to show the files available in the release. Please use the appropriate the PowerToys installer that matches your machine's architecture.  For most people, it is `x64`.
  
  - **For x64 processors (most common):** click on [PowerToysSetup-0.61.1-x64.exe](https://github.com/microsoft/PowerToys/releases/download/v0.61.0/PowerToysSetup-0.61.1-x64.exe)
- - **For ARM64 processors:** [PowerToysSetup-0.61.0-arm64.exe](https://github.com/microsoft/PowerToys/releases/download/v0.61.0/PowerToysSetup-0.61.0-arm64.exe)
+ - **For ARM64 processors:** [PowerToysSetup-0.61.1-arm64.exe](https://github.com/microsoft/PowerToys/releases/download/v0.61.0/PowerToysSetup-0.61.0-arm64.exe)
```

Comment:
Please edit the link as well for 0.61.1.
Since these are links I guess we can remove the click on mention.

Reply:
I've done that.
and since these are links, I think the description can also be removed.